### DISCUSSION TITLE
regolith-control-center not installed on ubuntu 20.04

### DISCUSSION THREAD

**Discussion by stefanofornari** [regolith-linux/regolith-desktop#768] [General]
Hello,
I installed Regolith 2.1 on my zorin box and it did not install regolith-control-center. I tried to install it manually but I've got the error message below.

Any advice on how to fix it? Is it a bug I should file somewhere?

Many thanks in advance.

```
uname -a
Linux minidesktop 5.4.0-131-generic #147~18.04.1-Ubuntu SMP Sat Oct 15 13:10:18 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

cat /etc/regolith/version 
REGOLITH_VERSION=2.1

sudo apt install regolith-control-center
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 regolith-control-center : Depends: libc6 (>= 2.29) but 2.28-0ubuntu1+zorin4 is to be installed
                           Depends: libcolord2 (>= 1.4.3) but 1.3.3-2build1 is to be installed
                           Depends: libgnome-desktop-3-19 (>= 3.33.4) but it is not installable
                           Depends: libnm0 (>= 1.11.90) but 1.10.6-2ubuntu1.4 is to be installed
                           Depends: libupower-glib3 (>= 0.99.8) but 0.99.7-2ubuntu0.18.04.1 is to be installed
                           Depends: libwacom2 (>= 1.1) but 0.29-1 is to be installed
                           Depends: gnome-settings-daemon (>= 3.29) but 3.28.1-0ubuntu1.3 is to be installed
                           Depends: gsettings-desktop-schemas (>= 3.31.0) but 3.28.0-1ubuntu1 is to be installed
                           Recommends: rygel but it is not going to be installed or
                                       rygel-tracker but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

```

**Comment by kgilmer** [comment]
Hi @stefanofornari , simply due to lack of dev capacity, we only support Ubuntu and Debian distros at this time.  In any case, it looks like the version of Regolith you're attempting to install does not line up w/ the GNOME dependencies in your system.

**Reply by stefanofornari** [reply]
Hi Ken,
thanks for the reply. Zorin is basically ubuntu, so please refer to Ubuntu 20.04. BTW, I will try to reproduce the issue on Ubuntu 20.04 and report back.

**Comment by kgilmer** [comment]
In that case you may find better luck with a different `apt` URL.  Regolith is available on `focal` and `jammy` and we'll soon add `kinetic`.  Is this the URL you specified?  

```
deb "[arch=amd64] https://regolith-desktop.org/release-ubuntu-focal-amd64 focal main"
```

**Reply by stefanofornari** [reply]
Hi Ken, yes, that's the url. I'll check for another repo or hopefully soon Zorin will upgrade to Ubuntu 20.04. Thanks for your help and for this great software.

**Reply by jc00ke** [reply]
> and we'll soon add `kinetic`.

This was clutch for me... I needed to switch from `jammy` to `kinetic` after upgrading to 22.10 in the `apt` source to get `i3xrocks-time` installed.
